modules/serverless-rn-1-6-0.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-6-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.6.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-6-0_{context}"]
+== New features
+* {ServerlessProductName} 1.6.0 is available on {product-title} 4.3 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.13.1.
+* {ServerlessProductName} has been updated to use Knative `kn` CLI 0.13.0.
```

Comment:
```suggestion
* {ServerlessProductName} has been updated to use Knative `kn` CLI 0.13.1.
```

Reply:
kn's 0.13.1 dot release is in progress (should be done upstream by today/tomorrow) and accordingly we'll do downstream.